src/System.Data.SqlClient/tests/ManualTests/SQL/IntegratedAuthenticationTest/IntegratedAuthenticationTest.cs

```diff
@@ -24,7 +26,7 @@ public static void IntegratedAuthenticationTestWithConnectionPooling()
             TryOpenConnectionWithIntegratedAuthentication(builder.ConnectionString);
         }
 
-        [CheckConnStrSetupFact]
+        [ConditionalFact(nameof(IsIntegratedSecurityEnvironmentSet))]
```

Comment:
I think it would be good to check for whether Connection Strings have been set in the env AND if Integrated security variable is set.

Reply:
Addressed in the comment above.